swagger.json

```diff
@@ -4508,6 +4579,27 @@
             }
           }
         },
+        "HosDailyLogDetailsResponse": {
+            "type": "object",
+            "properties": {
+                "dayStartMs": {
+                    "type": "integer",
+                    "format": "int64",
+                    "description": "Unix epoch time (in ms) of time fo when this daily log starts",
+                    "example": 1462881998034
+                },
+                "shippingId": {
+                    "type": "string",
+                    "description": "Shipping Id, potentially multiple Ids delimited by a comma or space",
+                    "example": "1059984"
+                },
+                "trailerNumber": {
```

Comment:
The request body uses the key `trailerName`, but the response uses `trailerNumber`.